### PR TITLE
Cloudbeat to support processing of k8s cert files 

### PR DESCRIPTION
**Pull request by uri-weisman**
FS fetcher to watch after k8s certificate & key files.
Supports CIS benchmark Rules 1.1.20 & 1.1.21.

### CODE REVIEW

File: tests/commonlib/io_utils.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def exec_command(container_name: str, command: str, param_value: str, resource:
                 f"File {resource} does not exist or mount missing.")
 
         if command == 'chmod':
-            os.chmod(path=resource, mode=int(param_value))
+            os.chmod(path=resource, mode=int(param_value, base=8))
```
Comment (uri-weisman) [inline tests/commonlib/io_utils.py:107]: int() is removing the leading zero and as `os.chmod` is expecting an octal value it results with wrong permissions.
Example:
```
int('0664') = 644
os.chmod(filepath, 644)
```
Causing the file permissions to be:
`-w--wx--- (230)`

**Comment by uri-weisman** [comment]
Related:
- https://github.com/elastic/csp-security-policies/pull/101

**Review by oren-zohar — APPROVED** [review]
💯 💯 💯 💯

**Review by DaveSys911 — APPROVED** [review]
LGTM

**Pull request merged**